templates/firefox-ios.xliff

```diff
@@ -4187,6 +4363,26 @@ Private Tab</source>
         <source>New Tab</source>
         <note>On the main menu, the title for the action that will create a new, non-private, tab.</note>
       </trans-unit>
+      <trans-unit id="MainMenu.ToolsSection.AccessibilityLabels.FindInPage.v132" xml:space="preserve">
+        <source>Find in page</source>
+        <note>On the main menu, the accessibility label for the action that will bring up the Search menu, so the user can search for a word or a pharse on the current page.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.ToolsSection.AccessibilityLabels.Save.v132" xml:space="preserve">
+        <source>Save</source>
```

Comment:
Please give more context as to "Save" here so localizers can translate appropriately